test/test_evaluator.cpp

```diff
@@ -32,6 +32,14 @@ inline void checkIntValue(
   EXPECT_EQ(actual_value, expected_value);
 }
 
+inline void checkValue(
```

Comment:
Does it make sense to name this as `checkConstEvaluate`? Otherwise it is hard to tell what this is checking.